Makefile

```diff
@@ -22,9 +22,12 @@ deps-macos:
 
 clean: 
 	rm cairo_syscalls/*json
+	rm cairo_programs/*json
 
 compile_cairo:
 	cairo-compile cairo_syscalls/syscalls.cairo --output cairo_syscalls/syscalls.json && \
+	starknet-compile cairo_programs/fibonacci.cairo > cairo_programs/fibonacci.json && \
```

Comment:
create a starknet_programs folder and move all the programs that are compiled with starknet-compile there

Reply:
Even more, we need to compile every program using starknet-compile

Reply:
i move those that use starknet-compile. we can not compile all the programs with starknet because some of them do not have the directive `%lang` so those one are simple cairo programs that are need for other purposes.